src/snarky.d.ts

```diff
@@ -51,489 +53,767 @@ declare interface ProvablePure<T> extends Provable<T> {
    */
   sqrt(): Field;
 
   /**
-   * Serialize the {@link Field} to a string, e.g. for printing.
-   * This operation does NOT affect the circuit and can't be used to prove anything about the string representation of the Field.
+   * Serialize the {@link Field} to a string, e.g. for printing. If you try to print a {@link Field} without this function it will directly stringify the Field object, resulting in an unreadable output.
+   * 
+   * **Warning: This operation does NOT affect the circuit and can't be used to prove anything about the string representation of the {@link Field}. Please use it only during debugging.
+   * 
+   * ```ts
+   * const someField = Field(42);
+   * console.log(someField.toString());
+   * ```
+   * 
+   * @return A string equivalent to the string representation of the Field.
    */
   toString(): string;
+
   /**
-   * Serialize this instance of a {@link Field} to a bigint.
-   * This operation does NOT affect the circuit and can't be used to prove anything about the bigint representation of the Field.
+   * Serialize the {@link Field} to a bigint, e.g. for printing. If you try to print a {@link Field} without this function it will directly stringify the Field object, resulting in an unreadable output.
+   * 
+   * **Warning: This operation does NOT affect the circuit and can't be used to prove anything about the bigint representation of the {@link Field}. Please use it only during debugging.
+   * 
+   * ```ts
+   * const someField = Field(42);
+   * console.log(someField.toBigInt());
+   * ```
+   * 
+   * @return A bigint equivalent to the bigint representation of the Field.
    */
   toBigInt(): bigint;
-  /**
-   * Serialize this instance of a {@link Field} to a JSON string.
-   * This operation does NOT affect the circuit and can't be used to prove anything about the string representation of the Field.
-   */
-  toJSON(): string;
 
   /**
-   * Returns the size of this type.
+   * Serialize the {@link Field} to a JSON string, e.g. for printing. If you try to print a {@link Field} without this function it will directly stringify the Field object, resulting in an unreadable output.
+   * 
+   * **Warning: This operation does NOT affect the circuit and can't be used to prove anything about the JSON string representation of the {@link Field}. Please use it only during debugging.
+   * 
+   * ```ts
+   * const someField = Field(42);
+   * console.log(someField.toJSON());
+   * ```
+   * 
+   * @return A string equivalent to the JSON representation of the {@link Field}.
    */
-  sizeInFields(): number;
+  toJSON(): string;
 
   /**
-   * Serializes this data structure into {@link Field} elements.
+   * This function is the implementation of {@link Provable.toFields} in {@link Field} type.
+   * Serialize this {@link Field} element into an array of {@link Field} elements.
+   * You can use this array to calculate the {@link Poseidon} hash of a {@link Field}.
```

Comment:
Delete this line? It makes no sense to me. You calculate the Poseidon hash of a Field with `Poseidon.hash([x])`